docs/source/configuration/telemetry/instrumentation/instruments.mdx

```diff
@@ -209,14 +217,15 @@ telemetry:
 
 #### `value`
 
-The `value` of an instrument is the value which will be drawn from. This can be one of the following:
+The `value` of an instrument is the value which will be drawn from and is dependent on the stage that the instrument is defined at.
+This can be one of the following:
 
-* `duration` - the duration of the pipeline service.
-* `unit` - the number of times the pipeline service has been executed.
-* `custom` - a custom value extracted from the pipeline service. See [selectors](./selectors) for more information.
-* `event_duration` - the duration of an event in the pipeline service.
-* `event_unit` - the number of times an event in the pipeline service has been executed.
-* `event_custom` - a custom value extracted from the event in the pipeline service. See [selectors](./selectors) for more information.
+* `duration` - the duration of the pipeline service (router/supergraph/subgraph).
+* `unit` - the number of times the pipeline service has been executed (router/supergraph/subgraph/graphql).
+* `custom` - a custom value extracted from the pipeline service. See [selectors](./selectors) for more information. (router/supergraph/subgraph/graphql)
+* `event_duration` - the duration of an event in the pipeline service (supergraph).
+* `event_unit` - the number of times an event in the pipeline service has been executed (supergraph).
+* `event_custom` - a custom value extracted from the event in the pipeline service. See [selectors](./selectors) for more information (supergraph).
```

Comment:
Assuming the parens denote which lifecycle stage the `value` is available at, this would be cleaner as a table (value / description / lifecycle stage availability).

LMK if you'd prefer for me to reformat it and I can push a commit.

Reply:
Yes, go ahead with any reformatting.

Reply:
[51467a1](https://github.com/apollographql/router/pull/5472/commits/51467a162ea3cae08fa7433f8390b108429655d8)